spec/index.html

```diff
@@ -1237,6 +1237,101 @@ <h3>Datatype IRIs</h3>
       is suggested in [[SWBP-XSCH-DATATYPES]]. RDF implementations
       are not required to support either of these facilities.</p>
   </section>
+</section>
+
+<section id="section-fragID" class="informative">
+  <h2>Fragment Identifiers</h2>
+
+  <p>RDF uses <a>IRIs</a>, which may include
+    <span id="dfn-fragment-identifiers"><!-- obsolete term--></span>
+    <dfn data-lt="fragment identifier" class="no-export">fragment identifiers</dfn>,
+    as resource identifiers.
+    The semantics of fragment identifiers is
+    <a data-cite="rfc3986#section-3.5">defined in
+    RFC 3986</a> [[RFC3986]]: They identify a secondary resource
+    that is usually a part of, view of, defined in, or described in
+    the primary resource, and the precise semantics depend on the set
+    of representations that might result from a retrieval action
+    on the primary resource.</p>
+
+  <p>This section discusses the handling of fragment identifiers
+    in representations that encode <a>RDF graphs</a>.</p>
+
+  <p>In RDF-bearing representations of a primary resource
+    <code>&lt;foo&gt;</code>,
+    the secondary resource identified by a fragment <code>bar</code>
+    is the <a>resource</a> <a>denoted</a> by the
+    full <a>IRI</a> <code>&lt;foo#bar&gt;</code> in the <a>RDF graph</a>.
+    Since IRIs in RDF graphs can denote anything, this can be
+    something external to the representation, or even external
+    to the web.</p>
+
+  <p>In this way, the RDF-bearing representation acts as an intermediary
+    between the web-accessible primary resource, and some set of possibly
+    non-web or abstract entities that the <a>RDF graph</a> may describe.</p>
+
+  <p>In cases where other specifications constrain the semantics of
+    <a>fragment identifiers</a> in RDF-bearing representations, the encoded
+    <a>RDF graph</a> should use fragment identifiers in a way that is consistent
+    with these constraints. For example, in an HTML+RDFa document [[HTML-RDFA]],
+    the fragment <code>chapter1</code> may identify a document section
+    via the semantics of HTML's <code>@name</code> or <code>@id</code>
+    attributes. The <a>IRI</a> <code>&lt;#chapter1&gt;</code> should
+    then be taken to <a>denote</a> that same section in any RDFa-encoded
+    <a>triples</a> within the same document.
+    Similarly, fragment identifiers should be used consistently in resources
+    with multiple representations that are made available via
+    <a data-cite="webarch/#frag-coneg">content negotiation</a>
+    [[WEBARCH]]. For example, if the fragment <code>chapter1</code> identifies a
+    document section in an HTML representation of the primary resource, then the
+    <a>IRI</a> <code>&lt;#chapter1&gt;</code> should be taken to
```

Comment:
```suggestion
    [[WEBARCH]]. For example, if the fragment identifier, <code>chapter1</code>, identifies a
    document section in an HTML representation of the primary resource, then the
    <a>IRI</a>, <code>&lt;#chapter1&gt;</code>, should be taken to
```

Reply:
```suggestion
    [[WEBARCH]]. For example, if the fragment identifier <code>chapter1</code> identifies a
    document section in an HTML representation of the primary resource, then the
    <a>IRI</a> <code>&lt;#chapter1&gt;</code> should be taken to
```

Reply:
Looks like the comment is on an old commit; the current text includes "identifier".

Reply:
@gkellogg yes, the comment is on an old commit. Notice that there are actually two separate change suggestions here: one from @TallTed and one from me. The one from me is a copy of Ted's suggestion but without the added commas, because also here I think that adding the commas is not correct (exactly as in the other case).